### PR TITLE
Uncompress .tar.gz file with python function

### PR DESCRIPTION
**Pull request by Dgelemi**
Remove dependencies to tar program, and make the data loaders for examples compatible on windows.
Also, use native python function to do the work

### CODE REVIEW

**Comment by Dgelemi** [comment]
finally managed to find how to do PR :)

There is a lot more work to do. Once the function is shown to work, need to replace all the files under molnet\load_function that are using tar

**Comment by Dgelemi** [comment]
See thread #599

**Comment by Dgelemi** [comment]
How do I find why it failed? Thank you

**Comment by mwojcikowski** [comment]
Click details by the Travis CI check (https://travis-ci.org/deepchem/deepchem/jobs/280640860). I think it's the ICU issue, see #857.

File: deepchem/utils/__init__.py [hunk 1]
```diff
@@ -83,6 +85,25 @@ def download_url(url, dest_dir=get_data_dir(), name=None):
   urlretrieve(url, os.path.join(dest_dir, name))
 
 
+def untargz_file(file, dest_dir=get_data_dir(), name=None):
+  """Untar and unzip a .tar.gz file to disk.
+  
+  Parameters
+  ----------
+  file: str
+    the file to decompress
```
Comment (lilleswing) [inline deepchem/utils/__init__.py]: the filepath to decompress

**Comment by rbharath** [comment]
@Dgelemi Thanks for your first PR! Looks good :)

Will merge in once Travis is working. I think this is the ICU build issue; restarted travis on this PR to see if we can get it passing now the ICU fix is in.

**Comment by Dgelemi** [comment]
Thank you. 
If it works with ICU fix, I can move to change the other files that have tar call

**Comment by rbharath** [comment]
@Dgelemi Future PRs with fixes much welcome.

This PR looks to be passing. Congrats on your first accepted DeepChem PR!! LGTM

**Pull request merged**